### ISSUE TITLE
Base64 Decoding error when getting File

### ISSUE THREAD

**Issue by LeakyBucket** [softprops/hubcaps#227]
## 🐛 Bug description
When making a request to get a file from a repo I am repeatedly getting a base64 error.  I've tried both the `README.md` in the repo root and my real target, an issue template.

#### 🤔 Expected Behavior
I was expecting to get back a `File` struct that I could use to read and modify the template for creating a new issue.

#### 👟 Steps to reproduce
I am seeing my issue in a unit test for the following function:

```
pub fn fetch<C: Clone + Connect + 'static>(repo: Repository<C>, name: &str) -> Result<File, Error> {
    let mut runtime = Runtime::new()?;

    runtime.block_on(
        repo.content()
            .file(name)
    )
}
```

The test doesn't do anything other than setup the Github struct and then get a repo reference to send to my `fetch` function.

The error I am seeing is as follows:

```
error = Error(
    Codec(
        Error("invalid value: byte array, expected valid base64 character at offset 60", line: 1, column: 1066),
    ),
    State {
        next_error: None,
        backtrace: InternalBacktrace {
            backtrace: None,
        },
    },
)
```

#### 🌍 Your environment
I am running on OS X 10.14.6, with `rustc` 1.37.0

hubcaps version: 0.5.0

**Comment by jasonwhite** [comment]
I think this should be fixed by #215. Can you confirm if this is still an issue on the latest in `master`? All we should need is a new release to include the fix.

**Comment by LeakyBucket** [comment]
@jasonwhite Yep, looks like `master` downloads and decodes the template correctly.
